packages/accounts-controller/CHANGELOG.md

```diff
@@ -7,6 +7,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [11.0.0]
+
```

Comment:
Looks like #3899 also adds a new package-level export (https://github.com/MetaMask/core/pull/3899/files#diff-cacd62f731d7f6947fe841f6ba3441c7a16f764b699801b0c212e0e6c15879bfR66-R74):

```suggestion

### Added

- Add and export function `isCustodyKeyring`, which validates whether a keyring type is a custody keyring ([#3899](https://github.com/MetaMask/core/pull/3899))

```

Reply:
I mistakenly added it for this package at first, but this seems to be a new function on `keyring-controller`.

Reply:
Ah my bad. In that case no issues here.